crates/polars-expr/src/state/node_timer.rs

```diff
@@ -29,7 +29,19 @@ impl NodeTimer {
         let nodes = &mut data.0;
         nodes.push(name);
         let ticks = &mut data.1;
-        ticks.push((start, end))
+        ticks.push((
```

Comment:
Can we replace this whole inner with:


```rust
store_raw(
    start.duration_since(self.query_start),
    end.duration_since(self.query_start),
    name)


```

Reply:
made `store_raw` `store_duration` and now can (`record_raw_timings` is now on the hook to convert u64 to nanosecond durations).